rbac/management/notifications/notification_hanlders.py

```diff
@@ -0,0 +1,179 @@
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""Notification handlers of object change."""
+import logging
+
+from django.conf import settings
+from management.notifications.producer_util import NotificationProducer
+
+from api.models import Tenant
+
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+producer = NotificationProducer()
+
+
+def notify_all(event_type, payload):
+    """Notify all tenants."""
+    # To avoid memory overloaded, use iterator:
+    # https://docs.djangoproject.com/en/4.0/ref/models/querysets/#django.db.models.query.QuerySet.iterator
+    for tenant in Tenant.objects.exclude(tenant_name="public").iterator():
+        # Tenant name pattern is acct12345
+        account_id = tenant.tenant_name.split("acct")[-1]
```

Comment:
Good to know, will use that instead.

Oh, you are right. So after we migrated to org_id, we just need to update account_id_for_tenant(tenant), and ask notification team to pull by org_id? Let me ask them if they have a plan to do that

Reply:
Yeah, and depending on the ordering of the PRs going in, would need to be updated in https://github.com/RedHatInsights/insights-rbac/pull/683 (or vice versa) -- cc: @mjholder

Reply:
They have a plan to check org_id but not started yet, might take months for us to be able to emit org_id instead of account number. https://issues.redhat.com/browse/RHCLOUD-19071